src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,6 @@
+Microsoft.VisualStudio.ProjectSystem.VS.FSharp.FSharpProjectSelector
```

Comment:
It's unfortunate that these become public, but it's a requirement for COM visible objects. Maybe obsolete it to avoid consumption?

Reply:
Actually, I stand corrected maybe only needed for IDispatch, I see MigrateXprojProjectFactory is internal and its almost identical to this.

Reply:
Yes, I'm pretty sure `public` is only required for `IDispatch` interfaces.